mos_tests/functions/common.py

```diff
@@ -301,10 +301,12 @@ def delete_instance(nova_client, uid):
             sleep(1)
 
 
-def create_instance(nova_client, inst_name, flavor_id, net_id, security_groups,
-                    image_id='', block_device_mapping=None, timeout=5):
+def create_instance(nova_client, inst_list, inst_name, flavor_id, net_id,
```

Comment:
Misha, several days ago you said me, that my test don't delete instance if during instance creation status isn't ACTIVE and exception raises. For this case inst_list was added in  parameter's list for create_instance common function. I update and check tests in nova_tests.py file. Test can manage this list itself, but try-exept-finaly construction should be added in this common function for return id, or in all tests which used this function